lib/exception_hunter/error_creator.rb

```diff
@@ -61,6 +62,13 @@ def notify(error)
           ExceptionHunter::SendNotificationJob.perform_later(serialized_slack_notifier)
         end
       end
+
+      def hide_sensitive_values(error_attrs)
+        sensitive_fields = ExceptionHunter::Config.sensitive_fields
+        return error_attrs if sensitive_fields.blank?
```

Comment:
This is mentioned thrice, once in the config as defaults, another here and another inside the redacter